usaspending_api/settings.py

```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for usaspending_api project.
+
+Generated by 'django-admin startproject' using Django 1.10.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.10/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.10/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '&=pi7xiahctpu@=nsp9!9qec6k(jcn##)b@u&s*sx9%kp+m66o'
```

Comment:
This'll throw warnings when/if you integrate [`bandit`](https://github.com/openstack/bandit) in the future. One approach that works relatively well is to generate a new secret key (e.g. via `django.utils.crypto.get_random_string`) whenever the app is restarted. The downside to this approach is that your user will get logged out.